### DISCUSSION TITLE
[animation] saveLatex() | /animation/example/savelatex/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#953] [Comments]
https://yihui.org/animation/example/savelatex/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
I just fixed it. Thanks for the report!

> Originally posted on 2020-07-07 15:17:55

**Comment by friendly** [comment]
Since the title is "Insert animations into a LaTeX document..." a more useful (for me) example would be to show how it is used in a chunk in an `.Rmd` or `.qmd` document, rather than stand-alone PDF.

**Reply by yihui** [reply]
For animations in R Markdown, please refer to https://bookdown.org/yihui/rmarkdown-cookbook/animation.html
